packages/devtools_app/lib/src/inspector/inspector_tree_flutter.dart

```diff
@@ -340,22 +354,28 @@ class _InspectorTreeState extends State<InspectorTree>
     currentAnimateTarget = null;
   }
 
+  double get safeViewportHeight {
+    return _scrollControllerY.hasClients
+        ? _scrollControllerY.position.viewportDimension
+        : 1000.0;
```

Comment:
put this in a const with a name that describes what it is

Reply:
done